trainer.py

```diff
@@ -16,11 +16,10 @@
 from chabud.model import ChaBuDNet
 
 
-# %%
 def cli_main(
     save_config_callback=None,
     seed_everything_default=42,
-    trainer_defaults: dict = {"logger": False, "precision": "bf16-mixed"},
+    trainer_defaults: dict = {"logger": False},
```

Comment:
```suggestion
    trainer_defaults: dict = {"logger": False, "precision": "16-mixed"},
```

@srmsoumya, does your GPU supports 16-bit mixed precision training? Apparently `bf16` doesn't work with TinyCD on Pytorch 2.0 (got `RuntimeError: "upsample_bilinear2d_out_frame" not implemented for 'BFloat16'`, see https://github.com/pytorch/pytorch/issues/88536). But I've tested `16-mixed` and it should be ok.

Just asking because my 8GB GPU ran out of memory on 32-bit precision with a batch size of 8, but the model runs if I use `16-mixed` with a batch size of 4.